components/Footer.js

```diff
@@ -254,12 +254,7 @@ const Footer = () => {
             maxWidth="300px"
           >
             <Flex my="12px">
-              <Image
-                src="/static/images/opencollectivelogo-footer-n.svg"
-                alt="Open Collective logo"
-                height={28}
-                width={167}
-              />
+              <Image src="/static/images/opencollectivelogo-footer-n.svg" height={28} width={167} />
```

Comment:
If it's a logo where "Open Collective" is clearly visible, the alt should be "Open Collective". Especially here, otherwise the "Make your community sustainable." doesn't make sense without "Open Collective".